tasks/main.yml

```diff
@@ -46,6 +71,21 @@
   tags:
     - enablerepos
 
+- name: check if gluster_repos_hci_subscribe is set
+  fail:
+    msg: "Please provide HCI repositories to be attached"
+  when: gluster_repos_hci_subscribe is not defined
```

Comment:
gluster_repos_hci_subscribe - is this mandatory?

Reply:
removed gluster_repos_hci_subscribe.